config/locales/en/prison_views.yml

```diff
@@ -3,6 +3,8 @@ en:
     none: None
   layouts:
     application:
+      contact_banner_html: |
+        <a href='%{url}'>Contact us</a> with feedback or to let us know about a problem.
```

Comment:
Do we need to add a Welsh translation as well or is this for another ticket?

Reply:
No, this is for staff where we don't translate into Welsh